src/sql-parser/tests/testdata/scalar

```diff
@@ -238,121 +238,120 @@ EXTRACT(YEAR FROM d)
 ----
 Function(Function { name: Name(UnresolvedItemName([Ident("extract")])), args: Args { args: [Value(String("year")), Identifier([Ident("d")])], order_by: [] }, filter: None, over: None, distinct: false })
 
-# TODO: Fix, need some help to find out where, is it in Expr::Function in expr.rs?
-#parse-scalar roundtrip
-#EXTRACT(YEAR FROM d)
-#----
-#EXTRACT(YEAR FROM d)
+parse-scalar roundtrip
+EXTRACT(YEAR FROM d)
+----
+extract('year', d)
```

Comment:
This is wrong:
```
materialize=> select extract('year', 'foo');
ERROR:  Expected FROM, found comma
LINE 1: select extract('year', 'foo')
                             ^
```

Reply:
This serializes as `"pg_catalog"."extract"('year,' 'foo')` which works though. I'm still a bit unclear on why that differs from this. Maybe we serialize using the stable version?